test/banach/pagination_test.clj

```diff
@@ -0,0 +1,18 @@
+(ns banach.pagination-test
+  (:require [banach.pagination :as p]
+            [clojure.test :refer [deftest is]]
+            [manifold.deferred :as md]
+            [manifold.stream :as ms]))
+
+(deftest paginated->stream-test
+  (let [gets (atom 0)
+        get! (fn [[marker n]]
+               (is (= ::todo marker))
+               (swap! gets inc)
+               (md/success-deferred
+                {:results (map (partial + (* 10 n)) (range 10))
+                 :todo (when (< n 9) [::todo (inc n)])}))
+        s (p/paginated->stream get! [::todo 0] :todo :results)]
+    (is (< 0 @gets 10))
```

Comment:
Sorry - what I'm meaning to say, is that I don't understand why we should be asserting that 0 < n < 10 gets have been performed, instead of asserting that 10 gets have happened.